code/modules/clothing/under/miscellaneous.dm

```diff
@@ -11,6 +11,7 @@
 	icon_state = "harness"
 	worn_state = "harness"
 	item_state = "w_suit"
+	body_parts_covered = 0

```

Comment:
I believe you should technically use `FALSE` or `null` here.

Reply:
0 is correct